aten/src/ATen/core/UnsafeFromTH.h

```diff
@@ -0,0 +1,20 @@
+#include <ATen/core/Tensor.h>
+
+namespace at {
+
+inline Tensor unsafeTensorFromTH(void * th_pointer, bool retain) {
```

Comment:
maybe we can just put this in at::internal or at::detail or something?  I don't think anyone besides us should be calling this.